### PR TITLE
PR-3 Readme and Gif File Updates Teams2.1

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**

### CODE REVIEW

File: samples/bot-all-cards/csharp/BotAllCards/Images/9.ListCard.png
Comment (Pawank-MSFT) [inline samples/bot-all-cards/csharp/BotAllCards/Images/9.ListCard.png:1]: Please hide **"Sai Sai"** in the image

File: samples/bot-teams-authentication/nodejs/Images/3.AuthToken.png
Comment (Pawank-MSFT) [inline samples/bot-teams-authentication/nodejs/Images/3.AuthToken.png:1]: Should be need to hide some part of the token?

**Review by Pawank-MSFT — COMMENTED** [review]
small comments please fix.

File: samples/bot-all-cards/csharp/BotAllCards/Images/9.ListCard.png
Comment (Mohammed-MSFT) [reply samples/bot-all-cards/csharp/BotAllCards/Images/9.ListCard.png:1]: fixed

Comment (Mohammed-MSFT) [reply samples/bot-all-cards/csharp/BotAllCards/Images/9.ListCard.png:1]: Fixed

File: samples/bot-teams-authentication/nodejs/Images/3.AuthToken.png
Comment (Mohammed-MSFT) [reply samples/bot-teams-authentication/nodejs/Images/3.AuthToken.png:1]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
looks correct, Approving!

**Review by Prithvi-MSFT — APPROVED** [review]

**Pull request merged**